app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -26,6 +26,8 @@ def button
     # Handle Toolbar Policy Tag Button
     @refresh_div = "main_div" # Default div for button.rjs to refresh
     tag(self.class.model) if params[:pressed] == "#{params[:controller]}_tag"
+    assign_policies(ContainerImage) if params[:pressed] == "container_image_protect"
+    check_compliance_images if params[:pressed] == "container_image_check_compliance"
```

Comment:
@moolitayer, I'm guessing that what we really need is a larger UI effort across the board that addresses this.  Outside just the `containers_common_mixin`.

Sounds like @jrafanie is talking about having a button object that can encapsulate what happens when the button is pressed.  That's one idea to handle this.

We might also wanna look at if angular introduces other metaphors for this.

But, this is definitely bigger than this PR.  And, imo, bigger than just the containers code.